scene/animation/animation_tree.cpp

```diff
@@ -733,6 +734,8 @@ void AnimationTree::_notification(int p_what) {
 }
 
 void AnimationTree::set_animation_player(const NodePath &p_path) {
+	ERR_FAIL_NULL_MSG(!p_path.is_empty() && Object::cast_to<AnimationPlayer>(get_node(p_path)), "Nodepath does not point to an AnimationPlayer.");
```

Comment:
```suggestion
	if (!p_path.is_empty()) {
		ERR_FAIL_NULL_MSG(Object::cast_to<AnimationPlayer>(get_node(p_path)), "NodePath does not point to an AnimationPlayer.");
	}
```

A bit clearer (plus formatting).